portfolio/src/main/webapp/tasksApi.js

```diff
@@ -143,19 +100,15 @@ function drawImportMenu() {
 }
 
 function clearImportMenu() {
+  importMenuVisible = false;
+
+  const $toggleTasksButton = $('#connect-tasks-btn')[0];
```

Comment:
Is this a jquery variable? If not, I think it shouldn't start with $

Reply:
Done.